index.html

```diff
@@ -11,11 +11,78 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+    <body>
+        <nav class="nav-bar">
+            <div class="nav-list">
+                <ul class="nav-links">
+                    <li class="dark">Meet Karma</li>
+                    <li>How it Works</li>
+                    <li>Store</li>
+                    <li>Blog</li>
+                    <li>Help</li>
+                    <li>Login</li>
+                </ul>
+                <img class="logo" src="img/karma-logo.svg" alt="Logo Image" width="26">
+            </div>
+        </nav>
+        
+        <section class="introduction">
+            <header class="header">
+                <h1>Introducing Karma</h1>
+                <h3>Bring WiFi with you, everywhere you go.</h3>
+                <div class="buttons">
+                    <button>Learn More</button>
+                </div>
+            </header>
+        </section>
 
-    <!-- Add your HTML markup here -->
```

Comment:
Please remove comments like this as they will be rendered in the browser and look a bit messy when people look at the source code. You should try to avoid leaving html comments on finished work wherever possible.